src/backend/gitalytics_api/common.py

```diff
@@ -46,8 +47,14 @@ async def endpoint(token: SessionKey):
             .filter(dbm.Session.id == session_id) \
             .one_or_none()
 
-    if not session:
-        raise fastapi.HTTPException(fastapi.status.HTTP_401_UNAUTHORIZED)
+        if not session:
```

Comment:
```suggestion
        if session is None:
```
liest sich meiner meinung nach besser